tools/wpt/tests/test_wpt.py

```diff
@@ -186,6 +186,7 @@ def test_run_failing_test():
     failing_test = "/infrastructure/expected-fail/failing-test.html"
 
     assert os.path.isfile("../../%s" % failing_test)
+    assert os.getenv("TASKCLUSTER_ROOT_URL")
```

Comment:
This is expected to fail outside of Taskcluster and will need to be reverted.

However, it currently also fails on Taskcluster, which suggests that `TASKCLUSTER_ROOT_URL` isn't passed in.